packages/@aws-cdk/aws-apprunner-alpha/test/auto-scaling-configuration.test.ts

```diff
@@ -0,0 +1,73 @@
+import { Match, Template } from 'aws-cdk-lib/assertions';
+import * as cdk from 'aws-cdk-lib';
+import { AutoScalingConfiguration } from '../lib';
+
+let stack: cdk.Stack;
+beforeEach(() => {
+  stack = new cdk.Stack();
+});
+
+test('create an Auto scaling Configuration with all properties', () => {
+  // WHEN
+  new AutoScalingConfiguration(stack, 'AutoScalingConfiguration', {
+    autoScalingConfigurationName: 'MyAutoScalingConfiguration',
+    maxConcurrency: 150,
+    maxSize: 20,
+    minSize: 5,
+  });
+
+  // THEN
+  Template.fromStack(stack).hasResourceProperties('AWS::AppRunner::AutoScalingConfiguration', {
+    AutoScalingConfigurationName: 'MyAutoScalingConfiguration',
+    MaxConcurrency: 150,
+    MaxSize: 20,
+    MinSize: 5,
+  });
+});
+
+test('create an Auto scaling Configuration without all properties', () => {
+  // WHEN
+  new AutoScalingConfiguration(stack, 'AutoScalingConfiguration', {
+  });
```

Comment:
```suggestion
  new AutoScalingConfiguration(stack, 'AutoScalingConfiguration');
```
Already defaulting to `{}`